src/plots/plotGroup.ts

```diff
@@ -0,0 +1,33 @@
+module Plottable.Plots {
+  export class PlotGroup extends Components.Group {
+    public entityNearest(point: Point): PlotEntity {
+      let closestPlotEntity: PlotEntity;
+      let minDistSquared = Infinity;
+      this.components().forEach((plot: Plot) => {
+        let candidatePlotEntity = plot.entityNearest(point);
+        if (candidatePlotEntity == null) {
+            return;
+        }
+
+        let position = candidatePlotEntity.position;
+        let distSquared = Math.pow(position.x - point.x, 2) + Math.pow(position.y - point.y, 2);
```

Comment:
Yeah, preferably, our entityNearest algorithm would use the nearest point on the entity not just "entity.position". However, for the purposes of this PR, I think it's fine to use the same standard as the library.